src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -120,6 +121,7 @@ public GetTasksCompletableResponse<Task> searchForCompletableTasks(
         List<String> taskTypes = extractTaskTypes(evaluateDmnResult);
 
         if (taskTypes.isEmpty()) {
+            log.info("TaskTypes is empty");
             return new GetTasksCompletableResponse<>(false, emptyList());
         }
```

Comment:
There are also some logging improvements/additions for the `TaskSearchController.searchWithCriteriaForAutomaticCompletion` .  E.g. when no role assignments are found output the userId.  We could also output the caseId and eventId for every request.

Reply:
UserId is already logged in https://github.com/hmcts/wa-task-management-api/blob/6b290b8fc5cd2a75d26f171ca967c9272be32b04/src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/access/AccessControlService.java#L37 which is called in https://github.com/hmcts/wa-task-management-api/blob/22c70d10d6c0e7732da77f271e0f2f25bd0d1f6b/src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskSearchController.java#L110

`caseId` is already logged as well but I've added in `eventId`

Reply:
Logs updated in the Controller class so resolved.